iosApp/iosApp/ComponentViews/ActionButton.swift

```diff
@@ -13,21 +13,24 @@ struct ActionButton: View {
     enum Kind {
         case back
         case close
+        case clear
```

Comment:
question(non-blocking): It seems like ~most of the properties are different for the clear button from the other two - is there a benefit to keeping it an ActionButton? I wonder if there is a different composition of Views here that could help reusability, but definitely not blocking for this PR.

Reply:
Yeah, I felt weird about doing this, it's the exact same style as the clear button except half the size and different colors, so it seemed awkward to copy all of that into a new component, but it's also awkward to fit it into here. Hm, it might be ideal to add modifiers or props to let you recolor the circle and icon, then just resize the entire action button. Though then the accessibility label would also be off, I'm not sure if adding another label higher up overrides any lower down?